pandas/_libs/tslibs/period.pyx

```diff
@@ -2663,6 +2663,7 @@ class Period(_Period):
         One of pandas period strings or corresponding objects. Accepted
         strings are listed in the
         :ref:`offset alias section <timeseries.offset_aliases>` in the user docs.
+        If value is not already period-like, freq is required.
```

Comment:
are you sure this is true? e.g.
```python
In [1]: pd.Period('2020-01')
Out[1]: Period('2020-01', 'M')
```

Reply:
@MarcoGorelli thanks. Good point, it works with a string, it just errors if value is None or datetime. Changed the note to reflect that.

Reply:
thanks - looks fine for `None` though?
```python
In [3]: pd.Period(None)
Out[3]: NaT
```

Reply:
hi - is this still active? if so, could you address this comment please?

Reply:
Sorry, missed this comment. I believe it should be all set now